build.xml

```diff
@@ -1,129 +1,109 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this Ant script even needed anymore?

Reply:
I don't think so. Removed.

Reply:
I do not think so, at least according to the code styles from PSR1/PSR2 that I'm using. And I am seeing that Magento\Framework does not include a new line.